src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -6160,6 +6185,22 @@ namespace SurfaceGeometry {
 			}
 		}
 
+		// Change algorithm for Kiva foundaiton surfaces
+		for (auto& surf : Surface) {
+			if (surf.ExtBoundCond == KivaFoundation) {
+				surf.HeatTransferAlgorithm = HeatTransferModel_Kiva;
+				if ( ! any_eq( HeatTransferAlgosUsed, HeatTransferModel_Kiva ) ) { // add new algo
+					HeatTransferAlgosUsed.redimension( ++NumberOfHeatTransferAlgosUsed );
```

Comment:
Fine by me. You should do a search and replace for these guys. This is borrowed from an example.